.azure/gpu-pipeline.yml

```diff
@@ -24,10 +24,15 @@ jobs:
           docker-image: 'pytorch/pytorch:1.9.0-cuda11.1-cudnn8-runtime'
           agent-pool: 'lit-rtx-3090'
           torch-ver: '1.9.0'
-        'PyTorch - stable':
+        'PyTorch - LTS':
```

Comment:
nit: LTS sounds a bit confusing to me. Maybe `PyTorch - 1.13` or `PyTorch - 1`?

Reply:
You're right! just wanted to make it look beginner-friendly and less confusing :)